src/bentoml/_internal/frameworks/transformers.py

```diff
@@ -613,6 +613,7 @@ def import_model(
     name: Tag | str,
     model_name_or_path: str | os.PathLike[str],
     *,
+    with_tokenizer: bool = False,
```

Comment:
why do we need this? can't we just always save the tokenizer with the model once we do import_model?

I think that is actually more intuitive right, since `import_model` users coming from HF will think that it will actually import the repository from HF.

Reply:
The current work flow (in our documentations) is that we save the tokenizer and the model separately. We want this param to allow users to have the choice of saving the tokenizer into the same directory as the model weights

current workflow: 
```
import bentoml
from transformers import AutoTokenizer

hf_model_name = "sentence-transformers/all-MiniLM-L6-v2"
tokenizer = AutoTokenizer.from_pretrained(hf_model_name)

bentoml.transformers.import_model("all-MiniLM-L6-v2", hf_model_name)
bentoml.transformers.save_model("all-MiniLM-L6-v2-tokenizer", tokenizer)
```

Reply:
Ye I don't think that makes much sense. Since we are moving to the new save methods in 1.2 and, probably just need to save the tokenizer together with the model, then users can do 
```python

tokenizer = transformers.AutoTokenizer.from_pretrained(bentomodel.path)
```

Reply:
@aarnphm why originally we didn't save tokenizer with model? If there is some usage for this case, we can set `with_tokenizer=True` and keep the option of not saving the tokenizer. If there's no usage, then we can remove `with_tokenizer` and always save tokenizer with model

Reply:
I think we should always save tokenizer with model. Makes this similar to hugging face `from_pretrained` without actually loading the model into memory.

Reply:
I think we should continue to keep models and tokenizers separate. If users want to save them in the same model, they can use the `bentoml.models.create` API.

Reply:
Please do not merge this before syncing up with me.